### ISSUE TITLE
Undefined is not a function. Not sure what is happening here.

### ISSUE THREAD

**Issue by NanoDami** [AtomLinter/linter-javac#63]
So i am just editing some java files when this file suddenly pops up when i try to save my project. Everything get's saved, but i am not sure why this error message pops up.

```
TypeError: undefined is not a function
  at Object.module.exports.getProjectRootDir (/home/user/.atom/packages/linter-javac/lib/init.coffee:109:8)
  at Object.lint (/home/user/.atom/packages/linter-javac/lib/init.coffee:39:20)
  at promises.push.Promise.then._this.emitter.emit.linter (/home/user/.atom/packages/linter/lib/linter-registry.js:94:28)
  at /home/user/.atom/packages/linter/lib/linter-registry.js:93:25
  at Set.forEach (native)
  at LinterRegistry.lint (/home/user/.atom/packages/linter/lib/linter-registry.js:91:20)
  at /home/user/.atom/packages/linter/lib/linter.coffee:98:16
  at Emitter.module.exports.Emitter.emit (/usr/local/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at /home/user/.atom/packages/linter/lib/editor-linter.js:47:28
  at Emitter.module.exports.Emitter.emit (/usr/local/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.saveAs (/usr/local/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:996:20)
  at TextBuffer.module.exports.TextBuffer.save (/usr/local/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:978:19)
  at TextEditor.module.exports.TextEditor.save (/usr/local/share/atom/resources/app.asar/src/text-editor.js:579:26)
```

**Comment by florianb** [comment]
Thanks for reporting @NanoDami - we're going to investigate that. It might be possible that we get back to you for further questions on that issue.. :coffee:

**Comment by Arcanemagus** [comment]
I'm betting that the issue here is that the file you were working on was opened as a "raw" file, instead of opening the directory in Atom as a "project".

**Comment by florianb** [comment]
This issue is probably related to #68 - i close this issue here for now. Hoping it get's resolved by fixing #68. :godmode:
